include/mbgl/osx/MGLMapView.h

```diff
@@ -184,11 +184,25 @@ IB_DESIGNABLE
 
 /** The minimum zoom level that can be displayed by the receiver using the
     current style. */
-@property (nonatomic, readonly) double maximumZoomLevel;
+@property (nonatomic) double maximumZoomLevel;
+
+/**
+ * Changes the maximum zoom level of the map.
+ *
+ * @param maximumZoomLevel The new minimum zoom level.
+ */
+- (void)setMaximumZoomLevel:(double)maximumZoomLevel;
```

Comment:
These changes aren’t necessary. By removing the `readonly` keyword from the property, you implicitly added a `-setMaximumZoomLevel:` method to the class.